### ISSUE TITLE
WCAG 2.0 and Children's Accessibility

### ISSUE THREAD

**Issue by agent508** [mholzschlag/WebStandardsProjectV.2#3]
I’d like to vent my frustration with the WCAG 2.0 success criteria and its widely adopted go-to as an industry standard. A long time ago I was invited to participate in the W3C working group to asses its guidelines as to children’s accessibility and Flash. Excited about this prospect initially, I was jostled between several different administrators, and ended up never been given access. The administration and form and function of the group seemed sufficiently lacking and I ended up stepping out. Since then I have been a part of several other standard groups, including the company accessibility standards by Pearson Ed and our own Flash specific standards, that have since been adopted by companies such as Adobe.    

As a web developer and standards advocate on children’s web accessibility, my work for several educational publishers such as Pearson and Harcourt school have led me to fill out numerous obligatory WCAG checklists. The WCAG checklists were frequently used to satisfy the priority I criteria, which left many websites severely lacking in functional accessibility.  Rules and guidelines were not written per use case, and as a result, many developers that were communicated the criteria during the retrofitting stages, did not know how to match them up to corresponding test cases.  In addition, no rules were specifically set for children’s access. The result was frequent misunderstanding, implementation errors and a general lack of proper integration.  

I would like to blow some new life into children’s accessibility advocacy, and introduce some measures that are age specific, for our younger web users. Our project, Mission Accessible, and our main character Agent 508, aims to bring back a focus to our younger web users who could benefit from a different (or expanded) set of accessibility rules and guidelines to make webpages geared towards children more accessible. We are looking into starting up a web initiative that allows for children to review certain websites on accessibility and provide feedback to developers. I would also like to see a list of new web accessibility standards that are organized by use case, in general. I will add several ideas here over the course of the next few weeks.
